src/main/java/org/apache/sling/scripting/core/impl/bundled/AbstractBundledRenderUnit.java

```diff
@@ -126,7 +124,7 @@ public BundleContext getBundleContext() {
                 }
             }
         }
-        return null;
+        return result;
```

Comment:
Should we remove the `return` at line 123 and have only one `return` statement? It would make the code simpler to read.

Reply:
It's a little bit a matter of taste IMHO. I personally find it easier to read when a method returns straight away when it can, rather than reading through the rest of the method to see that nothing else is happening before it returns at the end.
So therefore my preference would be to keep the 2 return statements as we have them now.
If you feel strongly about this one we can address it in a subsequent commit.